autoload.php

```diff
@@ -17,17 +17,28 @@
 
 if (file_exists('vendor/autoload.php')) {
     // prefer the composer autoloader
-    return require_once('vendor/autoload.php');
+    require_once('vendor/autoload.php');
+} elseif (!class_exists('TinCan\\Version')) {
```

Comment:
[PSR-2](http://www.php-fig.org/psr/psr-2/#5-1-if-elseif-else) is the php standard for code-style.

If a namespaced class is not recognized, then register the custom autoload function.  `Version` was chosen because it's likely the most ubiquitous.